test/e2e/main_test.go

```diff
@@ -124,6 +124,9 @@ func TestQueryPrometheus(t *testing.T) {
 		}, {
 			query:   `up{job="crio"} == 1`,
 			expectN: 1,
+		}, {
+			query:   `ALERTS{alertname="Watchdog"} == 1`,
```

Comment:
This only tests, whether rules from the same namespace are loaded, right?

Reply:
What does "same" namespace mean, when you request "all" namespaces? Sure the object is in the openshift-monitoring namespace, but the Prometheus object doesn't know that at selection time

Reply:
Added another e2e test that creates a new, non `openshift-monitoring` namespace, that should be picked up from.